django_file_form/forms.py

```diff
@@ -88,3 +89,16 @@ def get_placesholders_for_field(self, field_name):
         initial_values = get_list(self.initial.get(field_name, []))
 
         return [value.get_values() for value in initial_values if getattr(value, 'is_placeholder')]
+
+
+    def add_metadata_inputs(self):
+        for field_name in self.file_form_field_names():
+            metadata_field_name = f'{field_name}-metadata'
+            self.add_hidden_field(
+                metadata_field_name,
+                json.dumps(self.get_metadata_for_field(field_name))
+            )
+
+    def get_metadata_for_field(self, field_name):
+        initial_values = get_list(self.initial.get(field_name, []))
+        return {value.name: value.metadata for value in initial_values}
```

Comment:
```
{value.name: value.metadata for value in initial_values if value.metadata is not None} 
```
to avoid passing a bunch of filenames without metadata in a majority of cases?

Reply:
I updated this